content/en/hub/truecommand/admins/alerts.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Viewing and Configuring Alerts"
+linkTitle: "Alert System"
+description: "How to manage alerts and configure services to send alerts beyond the TrueCommand interface."
+---
+
+TrueCommand alerts provide visual warnings for monitored systems that require attention.
+These alerts are either generated by the monitored system or an alert rule created in TrueCommand.
+
+## Viewing All Alerts
+
+To see all alerts that TrueCommand has discovered, open the **Configure (Gear)** and click **All Alerts**.
+Administrator accounts can see all generated alerts.
+A non-administrator account can only view alerts according to their team and user account permissions.
+
+<img src="/images/tc-1.3-alert-notices.PNG">
+<br><br>
+
+The **Active Alerts** tab shows all unresolved alerts.
+Alerts are moved to the **Resolved Alerts** tab by clicking **Resolve (Check)**.
+To resolve multiple alerts, select each alert and click **Resolve (Check)**.
+
+Click **View alert notice details (`i`)** to view any user comments about an alert.
+
+Administrator accounts can delete an alert by clicking **Delete (Trash Can)**.
+Deleting an alert cannot be undone.
+To delete multiple alerts, select each alert and click **Delete (Trash Can)**.
+
+### Viewing Alerts by System
+
+Alerts generated by a monitored system display in both the administrative **Systems** screen and the **Dashboard** as a number above the system icon.
+To view all alerts for to a single system, go to the **Dashboard**, select a single system, and click the **Alerts** tab.
+
+<img src="/images/tc-1.3-alert-system.JPG">
+<br><br>
+
+Each active and resolved alert for that system is visible in this tab.
+Clicking **View alert notice details (`i`)** shows details for that alert, including the option to leave comments about the alert.
```

Comment:
This does not apply to the dashboard view of alerts.

Reply:
I'm confused.  This is still an option on the page.  
![image](https://user-images.githubusercontent.com/4654247/86361427-d32c1880-bc41-11ea-822d-105bacbe28b5.png)

Reply:
Sorry, I was under the impression this was the section talking about interacting with alerts on the dashboard - not the alerts page itself. The dashboard view of alerts has different buttons than the alerts page.